usb_autorunner_profiles/restore/run.sh

```diff
@@ -6,30 +6,124 @@ PWD=$(dirname "$0")
 REGISTRY_IP=10.0.90.99
 : "${NAMESPACE:=default}"
 
-JOB_NAME=openmrs-db-restore
+OPENMRS_JOB_NAME=openmrs-db-restore
+ODOO_JOB_NAME=odoo-db-restore
+OPENELIS_JOB_NAME=openelis-db-restore
+FILESTORE_JOB_NAME=filestore-restore
 OPENMRS_SERVICE_NAME=openmrs
 AUTORUNNER_WORKDIR=/opt/autorunner/workdir
-ARCHIVE_PATH=${AUTORUNNER_WORKDIR}/dump.sql
+ARCHIVE_PATH=${AUTORUNNER_WORKDIR}/archive
+DB_RESOURCES_PATH=${AUTORUNNER_WORKDIR}/db_resources
 
-mysql_app=`$kubectl get statefulsets.apps | grep 'mysql'`
-if [ -n "$mysql_app" ] ; then
-  echo "⚙️  Fetch MySQL credentials"
-  DB_USERNAME=`$kubectl get configmap mysql-configs -o json | jq '.data.MYSQL_ROOT_USER' | tr -d '"'`
-  DB_PASSWORD=`$kubectl get configmap mysql-configs -o json | jq '.data.MYSQL_ROOT_PASSWORD' | tr -d '"'`
-  DB_NAME=openmrs
+echo "🗂  Initialize local storage folders."
+# Create data volumes
+mkdir -p $SSD_MOUNT_POINT/data/postgresql
+mkdir -p $SSD_MOUNT_POINT/data/mysql
 
-  echo "Remove previous job, if exists"
-  $kubectl delete --ignore-not-found=true job ${JOB_NAME}
+# sync images to registry
+echo "⚙️  Upload container images to the registry at $REGISTRY_IP..."
+skopeo sync --scoped --dest-tls-verify=false --src dir --dest docker $PWD/images/docker.io $REGISTRY_IP
 
-  echo "Stop the OpenMRS service"
-  $kubectl scale deployment ${OPENMRS_SERVICE_NAME} --replicas 0
+echo "⚙️  Apply K8s description files"
+$kubectl apply -R -f $DB_RESOURCES_PATH
 
-  echo "⚙️  Run MySQL restore job"
-  cat <<EOF | $kubectl apply -f -
+echo "⚙️  Fetch MySQL credentials"
```

Comment:
Log does not match all lines.
Add the correct log lines for each steps (mysql root, psql, odoo, openelis...)